src/css/controls/flags/controls-hidden.less

```diff
@@ -10,6 +10,14 @@
         top: 0;
     }
 
+    .jw-media {
+        cursor: none;
+    }
+
+    &:not(.jw-flag-fullscreen) .jw-media {
+        cursor: default;
+    }
```

Comment:
Have you looked in "src/css/jwplayer/flags/controls-hidden.less"?

What isn't this handled there once and for all? Not specific enough? Looks like it should always be default.

Reply:
@robwalch  This is in that file -- 

The reason for the two rules is that, per your feedback on the last PR, my understanding was that we do not want a pointer in Fullscreen (this mirrors the behavior of setting ```controls: false``` in the configuration). So I wanted to ensure the cursor was visible in all instances, but fullscreen.

Reply:
"src/css/controls/flags/controls-hidden.less" does not equal
"src/css/jwplayer/flags/controls-hidden.less"

Reply:
@boyntoni Following up, the reason for the split in files is that `controls` is loaded only if `controls` are enabled. The code in `jwplayer` is always loaded as critical CSS. The fix will likely need to be addressed in both locations, or removed from `controls` completely since the styles may be cascading over each other.